docs/terms.md

```diff
@@ -0,0 +1,26 @@
+# What are the terms we use and what do they mean?
+
+### Group
+A group is a collection of unit files tied together. This can be seen as
+equivalent to kubernetes pods. Units within a Inago group are scheduled
+together on the same host.
+
+Since a group represents many units, Inago aims to abstract units away, so
+the user can manage a group more easily. Having this given, a group can be
+treated like a single unit. All actions one can apply to a unit using
+`fleetctl`, can also be applied towards a group using `inagoctl`.
+
+A group is named after the directory its unit files live in. Having two units
+`mygroup-foo@.service` and `mygroup-bar@.timer` in the directory `mygroup`,
+leads to the Inago group `mygroup`. So creating a directory and putting units
+into it, creates a new group as Ingao is able to understand it.
+
+### Slice
+A slice is a scalable group instance. The kubernetes equivalent would be a
+replica. Slices are a scaled version of a group. So you can scale up your group
+using slice expansion. See also [What is slice expansion?](slice_expansion.md).
+
+### Instance
+A group instance is a group in a fleet cluster. Having a group definition on
+your local file system and deploying this using Inago to a fleet cluster,
+creates a group instance. This group instance may be one of multiple slices.
```

Comment:
IMO you always have a slice. No matter if there is a `@` or not. Once instance of a group is a slice for me. So I wonder why we need the term `instance` when we have `slice`. Why making a difference in wording only because the unit name changes? At the end you have something running on a fleet cluster that was defined by a scheme. The scheme, the template, that is the group on your local file system, is a blueprint to schedule something. This something is either a slice of an instance. Please lets keep it simple and use one word. Since we heavily used the term `slice` in the code, I vote to simply throw `instance` away.